components/unzipped/dashboard/DesktopAccount.js

```diff
@@ -168,24 +168,26 @@ const DesktopAccount = ({
     ...initialState
   })
 
-  useEffect(async () => {
-    await getCurrentUserData()
-    await getPaymentMethods(token)
-    await getBusinessDetails(undefined, token)
+  useEffect(() => {
+    const fetchData = async () => {
+      await getCurrentUserData()
+      await getPaymentMethods()
+      await getBusinessDetails(undefined)
+    }
+
+    fetchData()
   }, [])
 
   useEffect(() => {
-    if (url && url?.url !== initialUrl) {
-      router.push(url?.url)
+    const fetchBalanceData = async () => {
+      getAccountBalance()
     }
-  }, [url, router])
 
-  useEffect(() => {
+    fetchBalanceData()
     // Call getAccountBalance on component load
-    getAccountBalance(token)
     // Set up an interval to call getAccountBalance every 5 minutes
     const intervalId = setInterval(() => {
-      getAccountBalance(token)
+      getAccountBalance()
```

Comment:
Do you need to pass token here, or you will get invalid response?

Reply:
![image](https://github.com/Unzipped-Technologies/unzipped/assets/14105328/c93ed4d5-c107-40f2-b81a-8ff25379b645)
Now, we are getting Auth Token direct from Store, We do not need to pass token from there because we can get that token from Store.

Reply:
![image](https://github.com/Unzipped-Technologies/unzipped/assets/14105328/30111beb-42a8-4a48-9001-9257b5601909)
It's working fine.